src/twisted/mail/_except.py

```diff
@@ -116,11 +116,12 @@ def __init__(
             desirable, they will be attempted, otherwise the delivery will be
             failed.
         """
-        if isinstance(resp, str):  # type: ignore[unreachable]
-            resp = resp.encode("utf-8")  # type: ignore[unreachable]
+        if not TYPE_CHECKING:
```

Comment:
https://mypy-play.net/?mypy=latest&python=3.11&flags=warn-unreachable%2Cwarn-unused-ignores&gist=1d25cd8354a91ea3864b255cb43ba82f

Reply:
oh no this is find